app/controllers/signs_controller.rb

```diff
@@ -18,7 +18,10 @@ def show
 
   def autocomplete
     if params[:term].present?
-      render json: open("#{AUTOCOMPLETE_URL}?q=#{CGI.escape(params[:term])}&limit=10", &:read).split("\n")
+      render json: open(
```

Comment:
🐘 can probably be replaced with `URI` rather than building the URI with string interpolation.